Classes/common/HTTP/CDTURLSessionTask.m

```diff
@@ -18,6 +18,12 @@
 #import "CDTHTTPInterceptor.h"
 #import "CDTLogging.h"
 
+@interface CDTHTTPInterceptorContext ()
+
+@property (nullable, nonatomic, strong) NSURLResponse *response;
```

Comment:
OK, why not just make it read-write in the main header? I can't really see the danger of that, if the user wants to completely replace the response it's up to them